content/en/guides/get-started/commands.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Commands and Deployment
+description: Nuxt.js comes with a set of useful commands, both for development and production purpose.
+position: 4
+category: get-started
+csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/master/01_get_started/02_routing?fontsize=14&hidenavigation=1&theme=dark
+---
+
+Nuxt.js comes with a set of useful commands, both for development and production purpose.
+
+## Using in package.json
+
+You should put these commands in the `package.json`:
+
+```json
+"scripts": {
+  "dev": "nuxt",
+  "build": "nuxt build",
+  "start": "nuxt start",
+  "generate": "nuxt generate"
+}
+```
+
+Then, you can launch your commands via `yarn <command>` (example: `yarn dev`).
+
+## Development Environment
+
+To launch Nuxt in development mode with hot reloading:
+
+<code-group>
+  <code-block label="npx" active>
+
+```bash
+nuxt
```

Comment:
`nuxt` -> `npx nuxt`

Reply:
should we remove the npx block and just show yarn and npm like rest of docs?